Makefile

```diff
@@ -105,6 +105,14 @@ build-release-tarballs: ## Create a series of `.tar.gz` files in the BIN_DIR dir
 
 ##@ Test
 
+.PHONY: unit
+unit: ## Run unit tests with coverage.
+	cargo llvm-cov nextest --html --locked --workspace --all-features -E 'kind(lib)' -E 'kind(bin)' -E 'kind(proc-macro)'
```

Comment:
```suggestion
.PHONY: test-unit
test-unit: ## Run unit tests.
    cargo nextest run --locked --workspace --all-features -E 'kind(lib)' -E 'kind(bin)' -E 'kind(proc-macro)'

.PHONY: cov-unit
cov-unit: ## Run unit tests with coverage.
    source <(cargo llvm-cov show-env --export-prefix)
    cargo llvm-cov clean --workspace
    $(MAKE) test-unit
```

Reply:
`cov-unit` gives an error for me unfortunately :(

```sh
source <(cargo llvm-cov show-env --export-prefix)
/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `source <(cargo llvm-cov show-env --export-prefix)'
make: *** [cov-unit] Error 2
```

Reply:
I went with the following instead. Feel free to modify it :)

```Makefile
UNIT_TEST_ARGS := --locked --workspace --all-features -E 'kind(lib)' -E 'kind(bin)' -E 'kind(proc-macro)'

.PHONY: test-unit
test-unit: ## Run unit tests.
	cargo nextest run $(UNIT_TEST_ARGS)

.PHONY: cov-unit
cov-unit: ## Run unit tests with coverage.
	cargo llvm-cov clean --workspace
	cargo llvm-cov nextest $(UNIT_TEST_ARGS)
```

Reply:
Also not 100% sure if `cargo llvm-cov clean --workspace` is needed. It looks like `cargo llvm-cov` already cleans old artifact by default.

```sh
$ cargo llvm-cov --help | grep clean
        --no-clean
            Build without cleaning any old build artifacts
```